hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/impl/ClientManagerImpl.java

```diff
@@ -153,6 +179,18 @@ public synchronized S3AsyncClient getOrCreateAsyncClient() throws IOException {
     return s3AsyncClient.eval();
   }
 
+  /**
+   * Get or create an unencrypted S3 client.
+   * This is used for unencrypted operations when CSE is enabled.
```

Comment:
nit: update doc to say it's only used when V1 compatibility is required because ...

Reply:
ack